tiles/src/main/java/com/protomaps/basemap/layers/PhysicalPoint.java

```diff
@@ -15,33 +20,213 @@ public String name() {
     return "physical_point";
   }
 
+  private static final double WORLD_AREA_FOR_70K_SQUARE_METERS =
+          Math.pow(GeoUtils.metersToPixelAtEquator(0, Math.sqrt(70_000)) / 256d, 2);
+
+  public void processNe(SourceFeature sf, FeatureCollector features) {
+    var sourceLayer = sf.getSourceLayer();
+    var kind = "";
+    var alkaline = 0;
+    var reservoir = 0;
+    var theme_min_zoom = 0;
+    var theme_max_zoom = 0;
+
+    if( sourceLayer.equals("ne_10m_lakes")) {
+      theme_min_zoom = 5;
+      theme_max_zoom = 5;
+
+      switch (sf.getString("featurecla")) {
+        case "Alkaline Lake" -> {
+          kind = "lake";
+          alkaline = 1;
+        }
+        case "Lake" -> kind = "lake";
+        case "Reservoir" -> {
+          kind = "lake";
+          reservoir = 1;
+        }
+        case "Playa" -> kind = "playa";
+      }
+
+      if (kind != "" && sf.hasTag("min_label") && sf.hasTag("name") && sf.getTag( "name") != null ) {
+        var water_label_position = features.pointOnSurface(this.name())
+                .setAttr("pmap:kind", kind)
+                .setAttr("pmap:min_zoom", sf.getLong("min_label") + 1)
+                .setZoomRange(sf.getString("min_label") == null ? theme_min_zoom : (int) Double.parseDouble(sf.getString("min_label")) + 1, theme_max_zoom)
+                .setBufferPixels(128);
+
+        NeNames.setNeNames(water_label_position, sf, 0);
+      }
+    }
+  }
+
   @Override
   public void processFeature(SourceFeature sf, FeatureCollector features) {
     if (sf.isPoint() && (sf.hasTag("place", "sea", "ocean") || sf.hasTag("natural", "peak"))) {
 
       // TODO: rank based on ele
 
+      String kind = "";
+
       int minzoom = 12;
-      if (sf.hasTag("natural", "peak")) {
-        minzoom = 13;
+      if (sf.hasTag("place", "ocean")) {
+        kind = "ocean";
+        minzoom = 0;
       }
       if (sf.hasTag("place", "sea")) {
+        kind = "sea";
         minzoom = 3;
       }
+      if (sf.hasTag("natural", "peak")) {
+        kind = "peak";
+        minzoom = 13;
+      }
 
       var feat = features.point(this.name())
         .setId(FeatureId.create(sf))
+        .setAttr("pmap:kind", kind)
```

Comment:
Move towards Tilezen standard of always exporting coallesed `kind` value from OSM tags